### PR TITLE
Lawnmowed

### PR DESCRIPTION
**Pull request by shurane**
Hey, I mowed your lawn for you. You're welcome. Could you merge this in, @jeremyandaya ?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<html>
+	<head>
+	</head>
+	<body>
+		<p>This is a paragraph tag.</p>
+		<div>This is a div</div>
+
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.js"></script>
+		<script>
+			console.log("I just loaded jQuery!");
```
Comment (jeremyandaya) [inline index.html:19]: What is this ehtesh???

Comment (shurane) [inline index.html:19]: Hey, you had an empty `index.html`. I'm just helping you out.

I might have some suggestions for `for loops` and stuff, too.

Comment (BanzaiBlazer) [inline index.html:19]: Leave the poor man alone, @shurane , mow your own lawn before i call the town.

Comment (jeremyandaya) [inline index.html:19]: Please add some while loops too when you're at it.

**Comment by troystarwalt** [comment]
I don't see enough divs.

**Comment by AlecStone** [comment]
horrible code.

**Comment by pmmurphy** [comment]
Get off his lawn.

**Comment by ChristineChan09** [comment]
What else can you do?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<!-- HTML documents start with...
```
Comment (troystarwalt) [inline index.html:2]: I usually don't write comments, but when I do they are full of sarcasm.

File: index.html (hunk 1 above)
Comment (adeshyne) [inline index.html:19]: gbogbo aye ti lo ohh...anybody english speak. my people!

**Comment by akatriel** [comment]
What's up with his < head >?

File: index.html (hunk 1 above)
Comment (joelsantos41) [inline index.html:19]: Why ruin this man's lawn @shurane.

**Comment by jeremyandaya** [comment]
nice code finally!!!

**Comment by jeremyandaya** [comment]
what took you so long??

**Pull request merged**